src/BaGet.Aliyun/BaGet.Aliyun.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="aliyun.oss.sdk.core" Version="2.5.3" />
```

Comment:
Is this the right package? Alibaba's [documentation](https://www.alibabacloud.com/help/doc-detail/32086.htm?spm=a2c63.l28256.b99.487.646c5139tDvkNm#title-uh3-q4n-6lr) indicates that [Aliyun.OSS.SDK](https://www.nuget.org/packages/Aliyun.OSS.SDK/) or [Aliyun.OSS.SDK.NetCore ](https://www.nuget.org/packages/Aliyun.OSS.SDK.NetCore/) are the recommended packages. This package seems to be a third-party alternative that is no longer maintained.

Reply:
Aliyun.OSS.SDK is for .net framework, Aliyun.OSS.SDK.NetCore is for .net core. But we need for .net standard. Aliyun go slow for .net because most of them use Java :-)

Reply:
Aliyun.OSS.SDK.NetCore seems to support .NET Standard: https://github.com/aliyun/aliyun-oss-csharp-sdk/blob/7efc8884e12bbc1accdc3fa1ee22d2bc3cec4717/sdk/aliyun-oss-sdk-dotnetcore.csproj#L12

Reply:
Yes, I checked the source code and already switched to that sdk.

The name of the SDK misleads me. Seems like they need pay more attention on .net :-)